src/components/FAB.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { StyleSheet, Share } from 'react-native';
+import { FAB } from 'react-native-paper';
+
+const styles = StyleSheet.create({
+  fab: {
+    position: 'absolute',
+    margin: 16,
+    right: 0,
+    bottom: 0
+  }
+});
+
+class SocialShare extends React.Component {
+  onShare = async () => {
+    const { userName, profileUrl } = this.props;
+    try {
+      const result = await Share.share({
+        message: `Check out this awesome developer @${userName}, ${profileUrl}`
+      });
+      return result;
+    } catch (error) {
+      alert(error.message);
+    }
+  };
+
+  render() {
+    return (
+      <FAB
+        style={styles.fab}
+        small
+        icon="add"
+        onPress={async () => await this.onShare()}
```

Comment:
@Chisom, kindly bear in mind that if you implement `async/await` in the `render` method, your component would await for any async execution on every single reconciliation, and the component tree would be stuck waiting on your component to finish downloading every time. So, do we really need to return anything within the `onShare` method?

Reply:
okay... make sense. I have corrected it